lib/managers/metadata.js

```diff
@@ -165,6 +165,24 @@ Metadata.prototype = {
 			};
 
 		return this.client.wrapWithDefaultHandler(this.client.put)(apiPath, params, callback);
+	},
+
+	/**
+	 * Delete a metadata template from an enterprise.
+	 *
+	 * API Endpoint: '/metadata_templates/:scope/:template/schema'
+	 * Method: DELETE
+	 *
+	 * @param {string} scope - The scope of the template to delete
+	 * @param {string} template - The template to delete
+	 * @param {Function} [callback] - Passed empty response body if successful, err otherwise
+	 * @returns {Promise<Object>} A promise resolving to the updated template
```

Comment:
This actually returns `{Promise<void>}` since there's no value coming back from the API